src/elements/content-sidebar/SidebarFileProperties.js

```diff
@@ -40,6 +40,7 @@ const SidebarFileProperties = ({
 }: Props) => (
     <LoadingIndicatorWrapper isLoading={isLoading}>
         <ItemProperties
+            archivedAt={Number(getProp(file, FIELD_METADATA_ARCHIVE)?.archiveDate) * 1000}
```

Comment:
Right now we plan to store unix epoch timestamps as strings in our template, but we're discussing changing their type to number.

Just in case we don't change it - is this conversion okay if we decide to store them as strings?

Reply:
Also I will need to make sure if they will be stored in seconds or miliseconds

Reply:
Our linter may complain about using a static built-in method like this. If so, use `parseInt(value, 10)` instead.

That said, we should handle formatting/display within `ItemProperties` rather than here, as well as handling non-number values better.

Reply:
But also - do we need something better than js `Date()` for handling this?

If so, I can try implementing something here, if you tell me which cases you need handled. 

But if not, shouldn't passing correct date format be responsibility of parent component using `ItemProperties`?

Reply:
I slightly refactored conversion here, now it passes undefined to prop when we don't have it metadata. My team finalized discussions and we'll have unix epoch timestamp in seconds stored as a string, so this should be correct conversion